### DISCUSSION TITLE
[Qiita CLI] 複数行にわたるタイトルの記事で末尾行以外が同じ内容の記事を複数pushした時の不具合

### DISCUSSION THREAD

**Discussion by zacky1972** [increments/qiita-discussions#588] [要望・アイデア・不具合]
たとえば，次のように複数行にわたるタイトルをつけていたとします．

```markdown
title: >-
  先頭行
  中間行
  末尾行
```

このような記事について，末尾行以外が同じ内容で，末尾行のみ異なる内容であるような記事を2つ書いたとします．

それを add/commit/push すると，なんと片方の記事が2つアップされてしまい，もう片方の記事が消えてなくなってしまう不具合があることに気づきました．

少なくとも，限定共有記事にしたときに，その現象が起こることを確認しました．

**Comment by tanaka-asami415** [comment]
@zacky1972 
ご報告ありがとうございます！

大変恐れ入りますが、タイトルを複数行で記載していただくことは想定をしていないため
意図していない動作が起きている可能性があります🙏

タイトルは1行で記載していただけると幸いです🙇
